### ISSUE TITLE
Headless moderation action logging/auditing

### ISSUE THREAD

**Issue by shiftyscales** [Neos-Metaverse/NeosPublic#2615]
## Is your tweak request related to a problem? Please describe.
@Nammi-namm and @Enverex have noted that when they have users set to moderator/admin, they don't have an easy/convenient way to list any moderation actions that occurred during any of the sessions the headless was hosting/which moderator invoked the action.

## Describe what would you like tweaked
A separate moderation log that prints moderation actions (silencing, kicks, bans, etc.) in a way that's easier to read/parse, listing the time of the action, the user that initated the action, and the user that was actioned.

## Describe alternatives you've considered
A command to print a list of moderation actions taken in a session. (Probably easiest to implement as a command that just prints the contents of the aforementioned log?

## Additional context
Nammi and Enverex were discussing headless hosting in-world, and I saw the potential use for a separate log file that is easier to parse/audit. After speaking with @glitchfur about this, they also felt it could be a useful addition in the future.

**Comment by shadowpanther** [comment]
By extension, an _audit_ log that lists user-related events like join/leave, role change (+ by whom), and all session-wide moderation actions (mute, kick, ban). Knowing that one headless can host multiple sessions, it also should list session ID for each action.
Some other actions might be also optionally added, like deleting the user root of another user (respawn guns).

**Comment by Frooxius** [comment]
Yeah this would definitely be good. Some of the stuff gets logged in into the main log but that's a bit complicated to parse and messy.

Ideally this seems like something that should be accessible to other users too? Although a basic version would just create a log  file for each session and print it out there for later.

Could we compile a list of the changes that would be good to have in this audit log?
